packages/backend-modules/discussions/graphql/schema.js

```diff
@@ -93,7 +93,15 @@ type mutations {
     targets: [CommentFeaturedTarget!]
   ): Comment!
 
-  suspendUser(id: ID!, until: DateTime, reason: String): User!
+  #
+  suspendUser(
+    id: ID!
+    #deprecated use numberDays instead
+    until: DateTime
+    @deprecated(reason: "Instead of inefficient date handling, we use numberDays")
```

Comment:
`@deprecated` is not a comment, but a declaration. Need to be next to deprecated field.

```suggestion
    until: DateTime @deprecated(reason: "Instead of inefficient date handling, we use numberDays")
```

Server and clients can log deprecated usage, graphiql will render them nice in docs.

<img width="363" alt="Bildschirmfoto 2022-11-18 um 09 33 33" src="https://user-images.githubusercontent.com/331800/202657644-996a0dd8-b7ea-4b7d-8d00-4b7dff2a9168.png">

<img width="374" alt="Bildschirmfoto 2022-11-18 um 09 34 14" src="https://user-images.githubusercontent.com/331800/202657672-d0d60962-766b-49d0-8de5-9d244ee60b2f.png">